src/components/Nav/Link.scss

```diff
@@ -0,0 +1,4 @@
+.navBar li a {
+    color: white;
```

Comment:
it's just white, but it should be still in variables.scss

Reply:
Also we are making dependency in here, in component we should keep scope of a component, so styles shouldn't depend on the .navBar class which is in other component. styling wouldn't work if we would like to put this component somewhere else, just add a class to <li>